cmd/serve.go

```diff
@@ -69,18 +79,42 @@ func cmdServe(cmd *cobra.Command, args []string) {
 		case <-ctx.Done():
 			return
 		case e := <-events:
-			switch e {
+			switch e.Type {
 			case kube.NewPodEvent:
-				metrics.GlobalPodMetrics.TotalNumberPods.Add(1)
+				podlog := log.New("pod", e.PodSelfLink)
+
+				cgPath := buildCgroupPath(cgroupRoot, e.PodUID, e.PodQOSClass)
```

Comment:
@asymmetric But that only gives me the resource URL I need the path to the pods cgroup here

@alban  I looked though http://godoc.org/k8s.io/api/core/v1#Pod to find a way to get the cgroup for pods but building it ourselfs seems to be the only way. Maybe @nhlfr knows...